core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -58,4 +135,30 @@ private FunctionResolver dayOfMonth() {
   private ExprValue exprDayOfMonth(ExprValue date) {
     return new ExprIntegerValue(getDateValue(date).getMonthValue());
   }
+
+  /**
+   * Time implementation for ExprValue.
+   * @param exprValue ExprValue of Time type or String.
+   * @return ExprValue.
+   */
+  private ExprValue exprTime(ExprValue exprValue) {
+    if (exprValue instanceof ExprStringValue) {
+      return new ExprTimeValue(getStringValue(exprValue));
```

Comment:
suggest to use exprValue.string() in future to get rid of the static getStringValue method.